docs/testing/troubleshooting.md

```diff
@@ -26,6 +26,25 @@ Logs do not affect the workbook.
 
 When your Excel Script encounters a problem running, it produces an error. You'll see a prompt pop-up asking if you want to **View Logs**. Press that button to open the console and display any errors.
 
+## Data limits
+
+There are hard limits set by the feature to prevent large amounts of data from being transferred at once and large numbers of individual transactions.
```

Comment:
```suggestion
Office Scripts enforces a limit on how much data can be transferred at once and how many individual transactions can be conducted. 
```
Not sure if this wording changes the meaning but a suggestion. I would avoid saying "the feature" (especially without naming it first) in docs but that might be my personal preference. It sounds PM-y to me.

Reply:
How about this?

"There are limits on how much Excel data can be transferred at once and how many individual Power Automate transactions can be conducted."

It attaches the actual controlling platform to the limit.